pkg/operator/resource/resourceapply/generic.go

```diff
@@ -371,6 +392,16 @@ func DeleteAll(ctx context.Context, clients *ClientHolder, recorder events.Recor
 	return ret
 }
 
+func DeleteAll(ctx context.Context, clients *ClientHolder, recorder events.Recorder, manifests AssetFunc, files ...string) []ApplyResult {
+	return DeleteAllWithPreprocess(ctx, clients, recorder, manifests, defaultPreprocessFunc(), files...)
+}
+
+func defaultPreprocessFunc() func(ctx context.Context, object runtime.Object) (runtime.Object, error) {
```

Comment:
naming suggestion: `nopPreprocess`

Reply:
I'd like to leave it as is. It's symmetric with `defaultPreprocessFunc` in `static_resource_controller.go`.